src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -220,10 +220,22 @@ details.undocumented > summary::before {
 	filter: invert(100%);
 }
 
-#crate-search, .search-input {
+.search-input {
 	background-color: #141920;
+	border-color: #424c57;
+}
+#crate-search {
+	background-color: #0f1419;
 	/* Without the `!important`, the border-color is ignored for `<select>`... */
-	border-color: #424c57 !important;
+	border-color: #5c6773 !important;
+	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.or\
```

Comment:
I gave a try to a svg minifier (just in case) and the result is indeed a bit shorter mostly because of the `<svg>` attributes:

```
data:image/svg+xml;utf8,<svg width="128" height="128" enable-background="new 0 0 128 128" version="1.1" viewBox="-30 -20 176 176" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><g fill="none" stroke="%235c6773" stroke-linecap="square" stroke-miterlimit="10" stroke-width="12"><line x1="111" x2="64" y1="40.5" y2="87.499"/><line x1="64" x2="17" y1="87.499" y2="40.5"/></g></svg>
```

Reply:
Just realized but please put `background-image` into `rustdoc.css` to prevent copying it into multiple places.

Reply:
It's a different svg depending on the theme (the color is different). That's the reason I've put it inline in the first place, since despite some searching, I couldn't find a nice way to modify the color of the existing svg from CSS. I just copied the pre-existing file `down-arrow.svg` and modified the color. I'm open to better solutions. E. g. to use `filter`, which can change color, one would somehow need to make the arrow it's own element or pseudo-element, presumably involving some helper-`div`. I don't know, perhaps I could give such a restructure a try, but I'm really not all that familiar with html and css and this stuff in the first place, so it's quite a bit of effort.

Reply:
The status quo is particularly unclean because of the forced line breaks because the tidy check otherwise fails the CIt.

Reply:
Ah I see. I had to do that recently so I can maybe help you with that. Give me a few minutes. :smile:

Reply:
You can disable the tidy check on a line iirc. To be confirmed.

Reply:
I see. Sorry, misunderstood what you meant previously. Then yes, you need to use a pseudo element.

Reply:
In the latest commit, I use a filter like that now, with a `:after` pseudo-element and a `div`. I hope everything works (or at least is not in a badly broken state) in all browsers we want to support; I would appreciate if you were able to create an online version again so I (and perhaps others) can test this on more devices.